stairctl/src/main/resources/application.yml

```diff
@@ -7,3 +7,10 @@ stairctl:
   host: ${STAIRCTL_HOST:127.0.0.1}
   port: ${STAIRCTL_PORT:5432}
   dbname: ${STAIRCTL_DBNAME:stairwaylib}
+
+spring:
+  shell:
+    interactive:
```

Comment:
Spring shell 3.3 disables the interactive shell by default, so we need to set a property to enable it.

https://github.com/spring-projects/spring-shell/wiki/Spring-Shell-3.3-Release-Notes